src/Scan/RestScans.spec.ts

```diff
@@ -0,0 +1,130 @@
+import { CliInfo } from '../Config';
+import {
+  Discovery,
+  Module,
+  ScanConfig,
+  SourceType,
+  StorageFile
+} from './Scans';
+import { RestScans } from './RestScans';
+import { instance, mock, reset } from 'ts-mockito';
+import nock from 'nock';
+
+interface ResponseBody {
+  name: string;
+  module: unknown;
+  tests: Array<unknown>;
+  fileId: string;
+  discoveryTypes: Array<Discovery>;
```

Comment:
```suggestion
  discoveryTypes: Discovery[];
```

Reply:
Could you explaine the difference?

Reply:
There is no difference. It is just the preferred way to declare an array according to our style guide. Moreover, you should write code in accordance with conventions adopted in the project (code must be consistent)